nikola/data/themes/base/assets/css/ipython.min.css.map

```diff
@@ -1 +1 @@
CC,sBAAA;EAGA,gBAAA;;AAGJ,GAAG;;;AAIH,KAAK;EACD,sBAAA;EACA,kBAAA;EACA,cAAA;EACA,WAAA;;EAEA,wBAAA;;EAEA,mBAAA;EACA,kBAAA;;AAGJ,KAAK,UAAU;EACX,gBAAA;;AAGJ,CAAC;EACG,mBAAA;;AAGJ,GAAG;EACD,YAAA;EACA,iBAAA;EACA,UAAA;;AAHF,GAAG,oBAKD;EACI,cAAA;EACA,qBAAA;;AAEA,GATH,oBAKD,EAIK;EACG,cAAA;EACA,qBAAA;;ACjNV;EAEI,WAAA;;;AAFJ,cAGI;EAAI,kBAAA;;AAHR,cAII;EAAQ,iBAAA;;AAJZ,cAKI;EAAG,0BAAA;;AALP,cAMI;EAAO,0BAAA;;AANX,cAOI;EAAU,0BAAA;;AAPd,cAYI;EAAI,iBAAA;EAAmB,oBAAA;EAAsB,iBAAA;EAAmB,gBAAA;;AAZpE,cAaI;EAAI,iBAAA;EAAmB,oBAAA;EAAsB,iBAAA;EAAmB,gBAAA;;AAbpE,cAcI;EAAI,iBAAA;EAAmB,oBAAA;EAAsB,iBAAA;EAAmB,gBAAA;;AAdpE,cAeI;EAAI,eAAA;EAAiB,iBAAA;EAAmB,iBAAA;EAAmB,gBAAA;;AAf/D,cAgBI;EAAI,eAAA;EAAiB,iBAAA;EAAmB,iBAAA;EAAmB,gBAAA;EAAkB,kBAAA;;AAhBjF,cAiBI;EAAI,eAAA;EAAiB,iBAAA;EAAmB,iBAAA;EAAmB,gBAAA;EAAkB,kBAAA;;AAjBjF,cAoBI,GAAE;EAAc,mBAAA;;AApBpB,cAqBI,GAAE;EAAc,mBAAA;;AArBpB,cAsBI,GAAE;EAAc,mBAAA;;AAtBpB,cAuBI,GAAE;EAAc,eAAA;;AAvBpB,cAwBI,GAAE;EAAc,eAAA;;AAxBpB,cAyBI,GAAE;EAAc,eAAA;;AAzBpB,cA2BI,GAAE,IAAI;AA3BV,cA4BI,GAAE,IAAI;EAAgB,iBAAA;;AA5B1B,cA6BI;EAAI,gBAAA;;AA7BR,cA8BI,GAAG;EAAI,kBAAA;;AA9BX,cA+BI,GAAG,GAAG;EAAI,kBAAA;;AA/Bd,cAgCI;EAAI,mBAAA;;AAhCR,cAiCI,GAAG;EAAI,uBAAA;;AAjCX,cAkCI,GAAG,GAAG;EAAI,uBAAA;;AAlCd,cAmCI,GAAG,GAAG,GAAG;EAAI,uBAAA;;AAnCjB,cAqCI,GAAG,GAAG,GAAG,GAAG;EAAI,mBAAA;;AArCpB,cAsCI,EAAE;EAAM,eAAA;;AAtCZ,cAuCI,EAAE;EAAM,eAAA;;AAvCZ,cAyCI;EACI,YAAA;EACA,uBAAA;;AA3CR,cA8CI;EAAK,eAAA;;AA9CT,cAgDI;AAhDJ,cAgDS;EACD,SAAA;EACA,sBAAA;EACA,WAAA;EACA,eAAA;EACA,YAAA;;AArDR,cAwDI;EAAY,eAAA;;AAxDhB,cA0DI;EACI,iBAAA;EACA,kBAAA;EACA,YAAA;EACA,yBAAA;EACA,iBAAA;EACA,YAAA;EACA,eAAA;EACA,mBAAA;;AAlER,cAoEI;EACI,8BAAA;EACA,sBAAA;;AAtER,cAwEI;AAxEJ,cAwEQ;AAxER,cAwEY;EACJ,iBAAA;EACA,sBAAA;EACA,oBAAA;EACA,mBAAA;EACA,mBAAA;EACA,eAAA;EACA,YAAA;;AA/ER,cAiFI;EACI,iBAAA;;AAlFR,cAoFI,MAAM,GAAE,UAAU;EACd,mBAAA;;AArFR,cAuFI,MAAM,GAAE;EACJ,mCAAA;;AAxFR,cA0FI,EAAE;EAAS,eAAA;;AA1Ff,cA4FI;EAAG,gBAAA;;AA5FP,cA6FI,EAAE;EAAK,eAAA;;AA7FX,cA+FI;EACI,cAAA;EACA,iBAAA;EACA,kBAAA;;AAlGR,cAoGI,EAAE;EAAO,eAAA;;AApGb,cAsGI;AAtGJ,cAsGS;EACD,eAAA;EACA,YAAA;;AACA,cAHJ,IAGK;AAAD,cAHC,IAGA;EACG,eAAA;;AA1GZ,cA+GI;EAAQ,sBAAA;;AA/GZ,cAgHI,EAAE;EAAU,eAAA;;AChHhB,GAAG;;EVsBC,oBAAA;EACA,8BAAA;EACA,0BAAA;EAEA,iBAAA;EACA,2BAAA;EACA,uBAAA;EAEA,YAAA;EACA,sBAAA;EACA,kBAAA;;EAGA,aAAA;EACA,mBAAA;EACA,oBAAA;;AUlCJ,QAAmC;EAE/B,GAAG,UAAW,MAAK;IACf,aAAA;;;AAIR,GAAG;;EAEC,aAAA;EACA,YAAA;EACA,cAAA;EACA,kBAAA;EACA,gCAAA;EACA,WAAA;EXdA,sBAAA;EACA,2BAAA;EACA,8BAAA;;AWgBJ,CAAC,YAAY;EACX,qBAAA;EACA,iBAAA;EACA,kBAAA;;AAIE,EAAC,MAAO;AAAR,EAAC,MAAO;AAAR,EAAC,MAAO;AAAR,EAAC,MAAO;AAAR,EAAC,MAAO;AAAR,EAAC,MAAO;EACJ,mBAAA;;AAIR,UAAU,SAAU;EAChB,aAAA;;AAGJ,UAAU,SAAU;EAChB,gBAAA;EAKA,kBAAA;;AAGJ,UAAU,SAAU,eACnB;AADD,UAAU,SAAU,eACf;AADL,UAAU,SAAU,eACX;EACP,eAAA;;AAIF,UAAU,WAAY;EAClB,aAAA;;AAGJ,UAAW,UAAU;EACnB,0BAAA;EACA,YAAA;;AAGF;AACA;AACA;AACA;AACA;AACA;EACI,iBAAA;EACA,aCxBsB,8CDwBtB;;AAGJ;EAAe,iBAAA;;AACf;EAAe,iBAAA;;AACf;EAAe,iBAAA;;AACf;EAAe,eAAA;;AACf;EACI,eAAA;EACA,kBAAA;;AAEJ;EACI,eAAA;EACA,kBAAA","file":"ipython.min.css"}
```

Comment:
Amusingly, those are not my paths, but rather the paths of someone on the ipython project.

Will remove.